### PR TITLE
flyTo and easeTo options

### PR DESCRIPTION
**Pull request by mikelmaron**
updated README to mention that flyTo and easeTo options can be included in the location array

### CODE REVIEW

**Comment by jbranigan** [comment]
Thanks for adding this @mikelmaron. This looks good to me. Do you think an example block including a couple of those values would be helpful? It took me a minute to figure out where to put those additional options.

Something like:
```
            location: {
                center: [-113.72917, 48.58938],
                zoom: 12.92,
                pitch: 39.50,
                bearing: 36.00,
                speed: 0.2,
                curve: 1
            },
```

**Comment by mikelmaron** [comment]
@jbranigan where are you thinking for the example block? Inline within the configuration options section, or up above within the sample configuration?

File: README.md [hunk 1]
```diff
@@ -233,6 +233,7 @@ Note: items in bold are **required**.
     - **`pitch`**: Angle of the map view. `0` is straight down, and `60` is highly tilted.
     - **`bearing`**: Degrees of rotation clockwise from North (`0`). Negative values represent counter-clockwise rotation.
 - `mapAnimation`: Defines the [animation type](https://docs.mapbox.com/mapbox-gl-js/api/#map#jumpto) for transitioning between locations. This property supports 'flyTo', 'easeTo', and 'jumpTo' animations. If not specified, defaults to `flyTo`.
+    - flyTo and easeTo [options](https://docs.mapbox.com/mapbox-gl-js/api/map/#flyto-parameters) (`curve`, `maxDuration`, `minZoom`, `screenSpeed`, `speed`) can be included in the `location` array
```
Comment (jbranigan) [inline README.md:236]:
```suggestion
    - flyTo and easeTo [options](https://docs.mapbox.com/mapbox-gl-js/api/map/#flyto-parameters) (`curve`, `maxDuration`, `minZoom`, `screenSpeed`, `speed`) can be included in the `location` array, for example:
                ```location: {
                center: [-113.72917, 48.58938],
                zoom: 12.92,
                pitch: 39.50,
                bearing: 36.00,
                speed: 0.2,
                curve: 1
            },```
```

Comment (jbranigan) [reply README.md:236]: I think including it inline there would be good if that works for you, just to clarify what the "location array" is.

**Comment by mikelmaron** [comment]
thanks @jbranigan -- going to merge now

**Pull request merged**